starters/default/gatsby-ssr.js

```diff
@@ -4,4 +4,6 @@
  * See: https://www.gatsbyjs.com/docs/ssr-apis/
  */
 
-// You can delete this file if you're not using it
+exports.onRenderBody = ({ setHtmlAttributes }) => {
+  setHtmlAttributes({ lang: `en` })
```

Comment:
Should or have we documented this? (I've typically myself set the html attributes with the Helmet component, so it's possible that's still documented somewhere!)

The context: it's a reasonably common use case for folks, and if I recall, it's one thing that Lighthouse recommends too.

Reply:
https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/#editing-html-and-body

Reply:
Thank you!